modules/telco-reference-application-workload-characteristics.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-reference-application-workload-characteristics_{context}"]
+= DU reference application workload characteristics
+
+The reference application workload has the following characteristics:
```

Comment:
In the next section it is referred to (correctly) as the representative application workload. Maybe this helps tie them together?
```suggestion
The representative reference application workload has the following characteristics:
```